### DISCUSSION TITLE
edgedb部署在docker上的问题。

### DISCUSSION THREAD

**Discussion by Tk8and15** [geldata/gel#4711] [Q&A]
![image](https://user-images.githubusercontent.com/118907993/203714956-96c7ba73-39ef-4a6b-9356-48b76a74eb65.png)
![image](https://user-images.githubusercontent.com/118907993/203715038-70ba3af5-f1e8-436e-87f8-c35939fa3423.png)

**Comment by Tk8and15** [comment]
你好，为什么不能够初始化项目呢？

**Reply by fantix** [reply]
您好，中文讨论请移步 https://gitee.com/edgedb/devrel ；这边为了您能更快获取答复，还请使用英文。

看起来您是在 Docker 中使用 EdgeDB CLI？请贴一下操作步骤。问题在错误提示中已经说了，不能使用 root 用户来跑 Postgres，解决方案还要看操作目的或者使用场景是什么。

> pg_ctl: cannot be run as root

This explained why it wouldn't start. Are you trying to use the EdgeDB CLI inside a Docker container? Why not just use the [EdgeDB Docker image](https://hub.docker.com/r/edgedb/edgedb)?
